src/contexts/WalletConnectionProvider/previouslyConnectedProvider.tsx

```diff
@@ -1,11 +1,31 @@
 import { useLocalStorage, useWallet } from '@solana/wallet-adapter-react';
 import React, { useContext, useEffect } from 'react';
 
+const OLD_KEY_NAME = 'open-wallet-previously-connected';
+const NEW_KEY_NAME = 'unified-wallet-previously-connected';
```

Comment:
do you think we should ship dynamic key?

`<UnifiedWalletAdapter appName={'Jupiter'} />`
then, key name will be `${appName}-wallet-previously-connected`

Reply:
is fine I guess since local storage is isolated by origin. the consumer can know this key is from this library when they are using, it will create confusion when we prefix it with their app name